packages/core/src/util/txInspector.ts

```diff
@@ -270,57 +267,75 @@ export const withdrawalInspector: WithdrawalInspector = (tx) =>
   tx.body.withdrawals?.length ? BigIntMath.sum(tx.body.withdrawals.map(({ quantity }) => quantity)) : 0n;
 
 /**
- * Inspects a transaction for minted or burned assets.
+ * Matching criteria functor definition. Ths functor encodes a selection criteria for minted/burned assets.
```

Comment:
small typo. Please address it when suqashing, no need for a separate fixup on it.
```diff
- Ths
+ This